tasks/main.yml

```diff
@@ -2,6 +2,16 @@
     that: LOAD_BALANCER_SERVER_IP | ipaddr
     msg: "The LOAD_BALANCER_SERVER_IP variable must be an IP address, not a domain name."
 
+- name: add HAProxy PPA
+  apt_repository:
+    repo: "{{ HAPROXY_PPA }}"
+  when: HAPROXY_PPA
```

Comment:
This needs to be:

```yml
  when: HAPROXY_PPA != ""
```

Just tested it and it was failing for me otherwise with:

```
FAILED! => {"failed": true, "msg": "The conditional check 'HAPROXY_PPA' failed. The error was: template error while templating string: expected token 'end of statement block', got 'vbernat'. String: {% if ppa:vbernat/haproxy-1.8 %} True {% else %} False {% endif %}\n\nThe error appears to have been in '/home/mslm/dev/open-craft/ansible-secrets/deploy/roles/load-balancer/tasks/main.yml': line 5, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: add HAProxy PPA\n  ^ here\n"}
```

Reply:
@UmanShahzad  I have made this change and pushed it as an amended commit so avoid another squashing step.